src/jit/lower.cpp

```diff
@@ -2321,10 +2323,14 @@ void Lowering::InsertPInvokeMethodProlog()
     store->gtOp.gtOp1 = call;
     store->gtFlags |= GTF_VAR_DEF;
 
-    GenTree* insertionPoint = m_blockRange.FirstNonPhiOrCatchArgNode();
+    GenTree* insertionPoint = firstBlockRange.FirstNonPhiOrCatchArgNode();
+    if (insertionPoint == nullptr)
+    {
+        insertionPoint = firstBlockRange.EndExclusive();
```

Comment:
I agree. I've really been finding the naming confusing myself. Let's do that as a follow-up change.